ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/ParticipantPruningIT.scala

```diff
@@ -49,6 +48,74 @@ class ParticipantPruningIT extends LedgerTestSuite {
     }
   })
 
+  test(
+    "PRQueryLatestPrunedOffsets",
+    "It should be possible to query the latest pruned offsets",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(participant, alice)) =>
+    for {
+      (prunedUpToInclusive_initial, allDivulgencePrunedUpToInclusive_initial) <-
+        participant.latestPrunedOffsets()
+
+      // Create a dummy contract to move the initial ledger end forward
+      _ <- participant.create(alice, Dummy(alice))
+      // Capture the offset at which to prune
+      ledgerEndAfterCreate <- participant.currentEnd()
+
+      // Created another dummy contract to allow pruning at the previously captured ledger end
+      // TODO pruning: Remove this statement once pruning semantics allow pruning at the ledger end
+      _ <- participant.create(alice, Dummy(alice))
+
+      // Prune the ledger without divulgence
+      _ <- participant.prune(ledgerEndAfterCreate, pruneAllDivulgedContracts = false)
```

Comment:
You might need sth like this https://github.com/digital-asset/daml/blob/main/ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/ActiveContractsServiceIT.scala#L806
for it to work in Canton

Reply:
🤔 I think you are right, good reminder. Will check it out

Reply:
Added safe pruning guard